test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsControllerSpec.scala

```diff
@@ -1167,7 +1173,8 @@ class ReliefDetailsControllerSpec
             mockAuthWithNoRetrievals()
             mockGetSession(
               sessionWithReliefDetailsAnswers(
-                allQuestionsAnswered.copy(lettingsRelief = None)
+                allQuestionsAnswered
+                  .copy(privateResidentsRelief = Some(AmountInPence(20)), lettingsRelief = None, otherReliefs = None)
```

Comment:
it'd be clearer if the spec was updated to say something like "the user has not answered that question and the amount of private residents relief is greater them zero"

Reply:
Absolutely. Done.